core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,30 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
```

Comment:
In addition to pointing users to admin manual, could also suggest looking at docs for relevant accumulo properties.